### PR TITLE
Adding ip list-new feature

### PR DESCRIPTION
**Pull request by galamo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
 (function (){
     console.log("this function works")
+    console.log("this function works")
```
Comment (galamo) [inline index.js]: why did you add 2 many console.log messages

Comment (tomersto) [reply index.js:3]: hello

Comment (jonivid) [reply index.js]: hello

Comment (ItzikOSD) [reply index.js]: Hi from Itzik

Comment (ohadas88) [reply index.js]: hello

Comment (TomGagement) [reply index.js:3]: hello

Comment (avivtc) [reply index.js]: hello

Comment (mikelbts89) [reply index.js]: hi

Comment (roiroi123) [reply index.js]: ok ok

Comment (AdirAsulin) [reply index.js]: ok

Comment (tomereyal) [reply index.js]: ok
